protobuf-provider/src/test/java/io/confluent/kafka/schemaregistry/protobuf/ProtobufSchemaTest.java

```diff
@@ -679,6 +679,210 @@ public void testComplexCustomOptions2() {
     assertEquals(normalized, normalizedSchema.canonicalString());
   }
 
+  @Test
+  public void testComplexCustomOptions3() {
+    String schemaString = "syntax = \"proto3\";\n"
+        + "package com.example.mynamespace;\n"
+        + "import \"confluent/meta.proto\";\n"
+        + "import \"confluent/type/decimal.proto\";\n"
+        + "option java_package = \"io.confluent.kafka.serializers.protobuf.test\";\n"
+        + "option java_outer_classname = \"ComplexProto\";\n"
+        + "message SampleRecord {\n"
+        + "  int32 my_field1 = 2 [(confluent.field_meta) = {\n"
+        + "    value1: -2.0,\n"
+        /*
+        + "    value2: - 2.0,\n"
+        + "    value3: - "
+        + "      # comment"
+        + "      2.0,\n"
+        */
```

Comment:
Are these unwanted comments?

Reply:
Those are actually from the spec but don't work.  They're edge cases that the wire parser doesn't handle at the moment.

Reply:
Let me add a comment to that effect